src/Auth0.ManagementApi/Models/EmailVerificationIdentity.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Newtonsoft.Json;
+
+namespace Auth0.ManagementApi.Models
+{
+    /// <summary>
+    /// Represents the identity object that can be sent on requests to create an email verification ticket or job.
+    /// </summary>
+    public class EmailVerificationIdentity
+    {
+        /// <summary>
+        /// Identity provider name of the identity (e.g. google-oauth).
+        /// </summary>
+        [JsonProperty("provider")]
+        public string Provider { get; set; }
+        /// <summary>
```

Comment:
Nit: I'd prefer to see some whitespace/blank line in here, consistent with the formatting in other files.

Reply:
Fixed